### PR TITLE
Adding Mardatonne

### PR DESCRIPTION
**Pull request by yagizhan49**
This pull request was made by the Gatsby Branch Editor.

### CODE REVIEW

File: data/en/bots/462212977882562571.md [hunk 1]
```diff
@@ -0,0 +1,17 @@
...
+  owner: NeotiDev
+images: []
+link: >-
+  https://discordapp.com/oauth2/authorize?client_id=462212977882562571&scope=bot&permissions=8
+nsfw: false
+pagename: Mardatonne
+prefix: +
+support: NqMA6xC
+---
+<iframe src="https://mardatonne.tk"></iframe>
```
Comment (Andre601) [inline data/en/bots/462212977882562571.md]: I don't know, if it's a thing with the preview, but the iframe looks a bît weird.

File: data/en/bots/462212977882562571.md [hunk 2]
```diff
@@ -0,0 +1,17 @@
...
+description: This bot will let you do whatever you want
+github:
+  owner: NeotiDev
+images: []
+link: >-
+  https://discordapp.com/oauth2/authorize?client_id=462212977882562571&scope=bot&permissions=8
+nsfw: false
+pagename: Mardatonne
+prefix: +
+support: NqMA6xC
```
Comment (Andre601) [inline data/en/bots/462212977882562571.md]: You should provide the full link, like `https://discord.gg/NqMA6xC`
The site doesn't auto-add the link to the code

Comment (lepon01) [inline data/en/bots/462212977882562571.md]: You need the full link.

File: data/en/bots/462212977882562571.md [hunk 3]
```diff
@@ -0,0 +1,17 @@
+---
+avatar: >-
+  https://images-ext-1.discordapp.net/external/rEC-FKA-1pKc_0Vo_6yetpfcdRsNLQr__4jYtqusZLg/%3Fsize%3D2048/https/cdn.discordapp.com/avatars/462212977882562571/cef0139e279e65dfe5ddf7cd2675cfac.png
+cover: null
+custom_path: mardatonne
+description: This bot will let you do whatever you want
+github:
```
Comment (lepon01) [inline data/en/bots/462212977882562571.md]: If there's no code for the bot, there's no point for this.

**Comment by yagizhan49** [comment]
Looks like a moderator already fixed these.

**Comment by yagizhan49** [comment]
I still couldn't address the issue but it seems to disconnect Discord.
I'm not sure what is causing this

**Pull request merged**